src/global/interfaces/link-button-properties.ts

```diff
@@ -275,3 +275,15 @@ export function forwardHostEvent(
 
   nestedActionElement.dispatchEvent(copiedEvent);
 }
+
+/**
+ * Forwards a change on the nested input element to the host element.
+ * In this way an event triggered in the ShadowDOM can cross its boundary and can be listened on the host component.
+ */
+export function forwardInnerEventToHost(event: Event, host: HTMLElement): void {
```

Comment:
I'm not sure if this is the right place as it has nothing to do with link-button-properties

Reply:
Yes, moved to a new file 👍